Makefile

```diff
@@ -54,3 +55,17 @@ ut-circle: calico_mesos_builder.created dist/calico_isolator
 	--with-xunit --xunit-file=/circle_output/output.xml; RC=$$?;\
 	[[ ! -z "$$COVERALLS_REPO_TOKEN" ]] && coveralls || true; exit $$RC'
 
+builder-rpm:
+	cd $(WD)/packages && docker build -t calico-mesos-builder .
```

Comment:
The confusion caused by the `cd`ing everywhere is an issue in and of itself, and $WD is only adding to it. For instance, you can cut this down:
```
rpm:
	cd packages && docker build -t calico-mesos-rpm-builder .
	mkdir -p $(WD)/packages/rpms
	mkdir -p $(WD)/packages/srpms
	cd packages && docker run --name=calico-mesos-rpm-builder \
	 -v $(WD)/packages/rpms:/opt/rpms \
	 -v $(WD)/packages/srpms:/opt/srpms \
	 calico-mesos-rpm-builder bash -c 'cp -r /root/rpmbuild/SRPMS /opt/srpms; \
	 cp -r /root/rpmbuild/RPMS /opt/rpms'
	docker rm -f calico-mesos-rpm-builder
```

By removing a lot of the redundant pathing:
```
rpm:
	docker build -t calico-mesos-rpm-builder ./packages
	mkdir packages/rpms
	mkdir packages/srpms
	docker run --name=calico-mesos-rpm-builder \
	 -v packages/rpms:/opt/rpms \
	 -v packages/srpms:/opt/srpms \
	 calico-mesos-rpm-builder cp -r /root/rpmbuild/RPMS /opt/rpms
	docker rm -f calico-mesos-rpm-builder
```
(You'll need to test that command first but I believe it accomplishes the same thing and is much easier to read)